dashboard/app/models/level_group.rb

```diff
@@ -87,4 +88,135 @@ def pages
   def plc_evaluation?
     levels.map(&:class).uniq == [EvaluationMulti]
   end
+
+  # Surveys: How many students must complete a survey before any results are shown.
+  SURVEY_REQUIRED_SUBMISSION_COUNT = 5
+
+  # Surveys: Given a sublevel, and the known response to it, return a result object.
```

Comment:
That case does exist, but I'm not yet sure whether the code to process results as an anonymized survey will have much in common with the code to process that case.  Might do, just not yet sure...